README.md

```
@@ -1,85 +1,116 @@
-## Corelab Challenge:
+# Core Notes — To‑do List (Next.js(ReactTS) + NestJS + PostgreSQL + Docker)
+
+Aplicação full‑stack de lista de tarefas (to‑do) chamada Core Notes. Front‑end em Next.js (React + TypeScript + TailwindCSS), back‑end em NestJS com PostgreSQL, ambos com suporte a execução local (npm) e via Docker Compose.
+
+**Repositório**
+- Front-end: <a href="https://github.com/DavidBotelhoo/corelab-web-challenge">`corelab-web-challenge`</a>
+- Back-end: <a href="https://github.com/DavidBotelhoo/corelab-api-challenge">`corelab-api-challenge`</a>
+
+
+**Visão Geral**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)